cuizinart/frontend/public/security/email/welcome.html

```diff
@@ -1,7 +1,7 @@
 <p>Hi {{user.email}},</p>
 
 <p>
-You have successfully created an account on Cuizinart (https://cuiznart.io). 
+You have successfully created an account on Cuizinart (https://cuizinart.io). 
```

Comment:
```suggestion
You have successfully created an account on Cuizinart (http://cuizinart.io). 
```
To avoid certificate errors, please remove the "s" (cuizinart,io doesn't have https but it forwards to tuna.cs.uwaterloo.ca which does, so it's fine)